src/Raven.Studio/typescript/models/database/tasks/ongoingTaskKafkaEtlEditModel.ts

```diff
@@ -0,0 +1,35 @@
+﻿/// <reference path="../../../../typings/tsd.d.ts"/>
+import ongoingTaskQueueEtlEditModel = require("models/database/tasks/ongoingTaskQueueEtlEditModel");
+
+class ongoingTaskKafkaEtlEditModel extends ongoingTaskQueueEtlEditModel {
+
+    get studioTaskType(): StudioTaskType {
+        return "KafkaQueueEtl";
+    }
+    
+    constructor(dto: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskQueueEtlDetails) {
+        super(dto);
+    }
+
+    toDto(): Raven.Client.Documents.Operations.ETL.Queue.QueueEtlConfiguration { 
+        return super.toDto("Kafka");
+    }
+    
+    static empty(): ongoingTaskKafkaEtlEditModel {
+        return new ongoingTaskKafkaEtlEditModel(
+            {
+                TaskName: "",
+                TaskType: "QueueEtl",
+                TaskState: "Enabled",
+                TaskConnectionStatus: "Active",
+                Configuration: {
+                    EtlType: "Queue",
+                    Transforms: [],
+                    ConnectionStringName: "",
+                    Name: "",
+                },
+            } as Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskQueueEtlDetails);
```

Comment:
can we avoid as here?

Reply:
or it requires tons of weird stuff to be added?

Reply:
leaving this, as discussed